packages/v4/patternfly-docs/content/design-guidelines/components/timestamp/timestamp.md

```diff
@@ -0,0 +1,51 @@
+---
+id: Timestamp
+section: components
+---
+
+## Elements
+
+<img src="./img/timestamp-elements.png" alt="Timestamp with tooltip underline" width="352"/>
+
+1. **Day:** Spelled out or abbreviated format. [See Numerics guidelines for more information](https://www.patternfly.org/v4/ux-writing/numerics).
+2. **Date:** Can be spelled out, abbreviated form, or digit format. [See Numerics guidelines for more information](https://www.patternfly.org/v4/ux-writing/numerics).
+3. **Time:** Should be displayed in 12-hour or 24-hour ((HH:MM:SS). [See Numerics guidelines for more information](https://www.patternfly.org/v4/ux-writing/numerics).
+
+4. **UTC/Timezone:** Should always be displayed last, in shorthand form and be in 24-hour format.
+5. **Tooltip:** Dotted line displayed if there is tooltip available.
+
+## Usage
+### Use a timestamp when:
```

Comment:
Restructure this so that the H3 is "When to use", and then start with regular text saying "Use a timestamp when:" and a mini bulleted list... ie:

Use a timestamp when:
* You want to represent the exact date and time that an event occurred
* Users can go back and make use of past content.